README.md

```diff
@@ -85,6 +125,40 @@ console.log(spectral.run({ spec: 'oas2', target: myOAS }));
 //           light: false } } } ]
 ```
 
+## Example implmentations:
+
+- [Spectral Bot](), a GitHub pull request bot using the [Probot]() framework, built by Taylor Barnett
+
+## FAQs
+
+**I want to lint my OpenAPI Specification documents but don't want to implement Spectral right now.**
+
+No problem! A hosted version of Spectral comes **free** with the Stoplight platform. Sign up for a free account [here]().
+
+**What is the difference between Spectral and [Speccy](https://github.com/wework/speccy)?**
+
+With Spectral, lint rules can be applied to _any_ JSON object, not just OAS 3 documents. The rule structure is different between the two. Spectral uses [JSONPath](http://goessner.net/articles/JsonPath/) `path` parameters instead of the `object` parameters (which are OAS-specific). Rules are also more clearly defined (thanks to TypeScript typings) and now require specifying a `type` parameter. Some rule types have been enhanced to be a little more flexible.
+
+An example of this includes the ability to specify the object to be linted in the `path` parameter itself, instead of relying on rule-specific options to be applied.
```

Comment:
Can probably just remove this last bit. I'm not sure it adds anything, and is more confusing than anything else.